frontend/src/pages/index.js

```diff
@@ -1,23 +1,49 @@
-import { Typography } from "antd";
+import { useState, useEffect } from 'react';
 
-const { Title } = Typography;
 
 function Home() {
-  const [res, setRes] = useState("no response");
-  useEffect(() => {
-    async function getUsers() {
-      const res = await fetch("http://localhost:3001/user/3");
-      setRes(res);
-    }
-    getUsers();
-  }, [res]);
+  // const [res, setRes] = useState("no response");
+  // useEffect(() => {
+  //   async function getUsers() {
+  //     const res = await fetch("http://localhost:3001/user/3");
+  //     setRes(res);
+  //   }
+  //   getUsers();
+  // }, [res]);
   return (
     <div>
-      <Title>{res}</Title>
-      <h4>Response</h4>
-      {res}
+      <h1 id="title">FlexDJ</h1>
+      <h2>Explore your favorite music from Spotify, YouTube, and SoundCloud!</h2>
+      <div>
+        <SearchBox/>
+      </div>
+      <h2>Sign in to create playlists!</h2>
+      <Login></Login>
     </div>
   );
 }
 
+function SearchBox() {
+  return (
+      <input className="input" type="text" placeholder="Search for Music..."></input>
+  )
+}
+
+function Login() {
+  return (
+      <div>
+        <input className="input" type="text" placeholder="Username"></input>
+        <input className="input" type="text" placeholder="Password"></input>
```

Comment:
this is really good, we can pair and also go over how to incorporate some of AntD's input fields (which will allow us to validate text etc :))